messages/cache_setting/create.go

```diff
@@ -0,0 +1,31 @@
+package cachesetting
+
+const (
+	CreateUsage                          = "cache-setting"
+	CreateShortDescription               = "Creates a new Cache Setting configuration"
+	CreateLongDescription                = "Creates a Cache Settings configuration based on given attributes to be used in edge applications"
+	CreateFlagEdgeApplicationId          = "Unique identifier for an edge application"
+	CreateFlagName                       = "The Cache Settings configuration name"
+	CreateFlagIn                         = "Path to a JSON file containing the attributes of the Cache Settings configuration that will be created; you can use - for reading from stdin"
+	CreateOutputSuccess                  = "Created Cache Settings configuration with ID %d\n"
+	CreateHelpFlag                       = "Displays more information about the create subcommand"
+	CreateFlagBrowserCacheSettings       = "Configures the amount of time that the content is cached in the web browser"
+	CreateFlagQueryStringFields          = "Gives a list of query strings parameters to be considered in the Cache Settings configuration, that will segregate the cache to the same URL"
+	CreateFlagCookieNames                = "Distinguishes objects in the Azion cache by name/value of cookies"
+	CreateFlagCacheByCookies             = "Whether cache by cookies is active or not"
```

Comment:
Just a suggestion to keep the pattern for enabled/disabled properties.

```suggestion
	CreateFlagCacheByCookiesEnabled             = "Whether cache by cookies is active or not"
```